soc/nordic/common/CMakeLists.txt

```diff
@@ -7,6 +7,12 @@ zephyr_linker_sources_ifdef(CONFIG_ARM SECTIONS arm_platform_init.ld)
 
 zephyr_library_sources_ifdef(CONFIG_POWEROFF poweroff.c)
 
+if(CONFIG_SOC_SERIES_NRF54HX)
+  zephyr_library_sources_ifdef(CONFIG_PM       power.c)
+  zephyr_library_sources_ifdef(CONFIG_POWEROFF power.c)
+  zephyr_library_sources_ifdef(CONFIG_PM_S2RAM pm_s2ram.c)
+endif()
```

Comment:
please, place this code in the 54hx folder, not common, as you're making it 54hx specific anyway.

Reply:
done